src/app/profiles/App/Pages/index.tsx

```diff
@@ -3,18 +3,23 @@ import React from 'react';
 import Profiles from './Profiles';
 import Error from './Error';
 import Subscriptions from './Subscriptions';
-import { CONTRIBUTORS_PATH } from '../../routes';
+import { path } from '../../../routes';
+import useChangeLanguage from '../../../hooks/useChangeLanguage';
 
-const Pages = () => (
-  <>
-    <Switch>
-      <Redirect exact path="/" to={CONTRIBUTORS_PATH} />
-      <Redirect from={'/en/*'} to={CONTRIBUTORS_PATH} />
-      <Route path={CONTRIBUTORS_PATH} component={Profiles} />
-      <Route path="/mes-abonnements" component={Subscriptions} />
-      <Route component={Error} />
-    </Switch>
-  </>
-);
+const Pages = () => {
+  const lang = useChangeLanguage();
+  return (
+    <>
+      <Switch>
+        <Redirect exact path="/" to={path.fr.CONTRIBUTORS} />
+        <Redirect exact path="/en" to={path.en.CONTRIBUTORS} />
+        <Route path={path.fr.CONTRIBUTORS} component={Profiles} />
```

Comment:
Not sure it works, but what would happen if instead both routes, you only had one :
```
<Route path={t('profiles.paths.contributors')} component={Profiles} />
```
Would be awesome.


Maybe another possible solutions could be to nest routes:
<Route path="/:locale}" >
    <Route path={...} component={Profiles} />
</Route>

Reply:
i tried but it doesn't work, there is some link like this : `${path[lang].CONTRIBUTORS}/${contributor.id}/${slugify(
    contributor.name
  )}`;
with dynamique path like `:locale`, this `path[lang].CONTRIBUTORS` give: `/:locales/guides/13/lmem` instead of `/en/guides/13/lmen`